### ISSUE TITLE
 cmake mac os None of the required 'hidapi-libusb;hidapi'

### ISSUE THREAD

**Issue by windoozatnick** [dmage/co2mon#37]
Не собирается. При этом [вот это](https://github.com/dmage/co2mon/issues/31) видел, и вот так делал:
`brew install libusb
brew install hidapi
brew install pkg-config`

**Comment by dmage** [comment]
Что выдает команда `pkg-config hidapi --debug`?

**Comment by windoozatnick** [comment]
`pkg-config hidapi --debug`
`no output option set, defaulting to --exists`
`Error printing disabled by default due to use of output options --exists, --atleast/exact/max-version, --list-all or no output option at all. Value of --print-errors: 0`
`Error printing disabled`
`Adding virtual 'pkg-config' package to list of known packages`
`Looking for package 'hidapi'`
`Looking for package 'hidapi-uninstalled'`

**Comment by dmage** [comment]
Странно, после этих строк должны быть
```
Reading 'hidapi' from file '/usr/local/lib/pkgconfig/hidapi.pc'
Parsing package file '/usr/local/lib/pkgconfig/hidapi.pc'
```
Есть ли в систем файл `/usr/local/lib/pkgconfig/hidapi.pc`? Показывается ли он в `brew list hidapi`?

**Comment by windoozatnick** [comment]
Есть симлинк
`hidapi.pc -> ../../Cellar/hidapi/0.8.0-rc1/lib/pkgconfig/hidapi.pc`

На `brew list hidapi` выдает:
`/usr/local/Cellar/hidapi/0.8.0-rc1/bin/hidtest`
`/usr/local/Cellar/hidapi/0.8.0-rc1/include/hidapi/hidapi.h`
`/usr/local/Cellar/hidapi/0.8.0-rc1/lib/libhidapi.0.dylib`
`/usr/local/Cellar/hidapi/0.8.0-rc1/lib/pkgconfig/hidapi.pc`
`/usr/local/Cellar/hidapi/0.8.0-rc1/lib/ (2 other files)`
`/usr/local/Cellar/hidapi/0.8.0-rc1/share/doc/ (6 files)`

**Comment by dmage** [comment]
Да, у меня точно так же. Похоже что-то не так с pkg-config, возможно стоит посмотреть на вывод
```
$ pkg-config --variable pc_path pkg-config
/usr/local/lib/pkgconfig:/usr/local/share/pkgconfig:/usr/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig/10.12
```

**Comment by windoozatnick** [comment]
`$ pkg-config --variable pc_path pkg-config`
`/opt/local/lib/pkgconfig:/opt/local/share/pkgconfig`
